rajapinnat/logmaster/outbound_delivery.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+date_default_timezone_set('Europe/Helsinki');
+
+if (trim($argv[1]) == '') {
+  die ("Et antanut l�hett�v�� yhti�t�!\n");
+}
+
+// lis�t��n includepathiin pupe-root
+ini_set("include_path", ini_get("include_path").PATH_SEPARATOR.dirname(dirname(dirname(__FILE__))));
+ini_set("display_errors", 1);
+
+error_reporting(E_ALL);
+
+// otetaan tietokanta connect ja funktiot
+require "inc/connect.inc";
+require "inc/functions.inc";
+
+// Sallitaan vain yksi instanssi t�st� skriptist� kerrallaan
+pupesoft_flock();
+
+$yhtio = mysql_escape_string(trim($argv[1]));
+$yhtiorow = hae_yhtion_parametrit($yhtio);
+$kukarow = hae_kukarow('admin', $yhtio);
+
+if (!isset($kukarow)) {
+  exit("VIRHE: Admin k�ytt�j� ei l�ydy!\n");
+}
+
+if (!in_array($yhtiorow['ulkoinen_jarjestelma'], array('', 'K'))) {
+  die ("Ker�tt�vien tilauksien l�hett�minen estetty yhti�tasolla!\n");
+}
+
+$query = "SELECT *
+          FROM lasku
+          WHERE yhtio = '{$kukarow['yhtio']}'
+          AND (
+            (tila = 'N' AND alatila = 'A') OR
+            (tila = 'G' AND alatila = 'J')
```

Comment:
`lahetteen_tulostusjono.php`:n kysely on melkoisesti monipuolisempi. Sitä pitäisi katsoa hieman tarkemmin.

Ensisilmäyksellä sielä pisti silmään, että siirtolistoissa pitää olla myös `and tilaustyyppi != 'M'`.